### ISSUE TITLE
Add support for SPDX license expressions

### ISSUE THREAD

**Issue by teo-tsirpanis** [fsprojects/Paket#3475]
At the moment, Paket does not support the [`<PackageLicenseExpression/>` attribute](https://github.com/NuGet/Home/wiki/Packaging-License-within-the-nupkg-(Technical-Spec)#spdx-specification).

Instead, it crashes with the a message like this:

```
C:\code\.paket\Paket.Restore.targets(233,5): error : The licenseUrl and license elements cannot be used together. [C:\code\MyProject.fsproj]
```

**Comment by arthurrump** [comment]
I think this should be looked at, especially since [`<PackageLicenseUrl>` is being deprecated](https://docs.microsoft.com/nl-nl/dotnet/core/tools/csproj#packagelicenseurl):

    \.paket\Paket.Restore.targets(261,5): warning NU5125: The 'licenseUrl' element will be deprecated. Consider using the 'license' element instead.

The new way is to use the [`license`](https://docs.microsoft.com/en-us/nuget/reference/nuspec#license) element in the .nuspec with an attribute indicating the use of an SPDX license expression or a file in the package. These correspond with `<PackageLicenseExpression>` and `<PackageLicenseFile>` elements in the project file.

**Comment by teo-tsirpanis** [comment]
In Paket 2.215.0, the issue does not reproduce, for some reason, and adding a license expression works as intended.

Has it been fixed at some point? Should I close it?

cc. @matthid, @forki

**Comment by teo-tsirpanis** [comment]
Apparently, it only works for certain versions of the .NET Core SDK.

**Comment by teo-tsirpanis** [comment]
I think I got it. Between versions 2.2.203 and 2.2.301 of the .NET Core SDK, there has been a change in the code that raises this error (perhaps a bug fix?).

The assembly is `NuGet.Build.Tasks.Pack.dll`.

![image](https://user-images.githubusercontent.com/12659251/61234627-37f21880-a73c-11e9-840c-4583914c34f7.png)

So, it had been a problem of the .NET Core SDK that got fixed at some point. what remains is to [have AppVeyor catch up](https://github.com/appveyor/ci/issues/3028).
